util/reconciler/defaults.go

```diff
@@ -66,14 +66,10 @@ const (
 	// DefaultDNSTTLSec is the default TTL used for DNS entries for api server loadbalancing
 	DefaultDNSTTLSec = 30
 
-	// DefaultLinodeTooManyPOSTRequestsErrorRetryDelay is the default requeue delay if there is Linode API error for POST request. Currently, it is set to 10 requests per 30 seconds
-	DefaultLinodeTooManyPOSTRequestsErrorRetryDelay = 30 * time.Second
-	// SecondaryLinodeTooManyPOSTRequestsErrorRetryDelay is the secondary requeue delay if there is Linode API error for POST request. Currently, it is set to 5 requests per 15 seconds
-	SecondaryLinodeTooManyPOSTRequestsErrorRetryDelay = 15 * time.Second
-	// DefaultPOSTRequestLimit is the default limit of how many POST requests can be made to /linode/instances endpoint in 30 seconds before rate-limit reset.
-	DefaultPOSTRequestLimit = 10
-	// SecondaryPOSTRequestLimit is the secondary limit of how many POST requests can be made to /linode/instances endpoint in 15 seconds before rate-limit kicks in.
-	SecondaryPOSTRequestLimit = 5
+	// DefaultLinodeTooManyPOSTRequestsErrorRetryDelay is the default requeue delay if there is Linode API error for POST request
+	DefaultLinodeTooManyPOSTRequestsErrorRetryDelay = 15 * time.Second
```

Comment:
can this be taken directly from headers instead of hard coded here?

Reply:
We don't have this in the headers unfortunately. Here is a sample header:
```
[Access-Control-Allow-Credentials:[true] Access-Control-Allow-Headers:[Authorization, Origin, X-Requested-With, Content-Type, Accept, X-Filter] Access-Control-Allow-Methods:[HEAD, GET, OPTIONS, POST, PUT, DELETE] Access-Control-Allow-Origin:[*] Access-Control-Expose-Headers:[X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Status] Cache-Control:[max-age=0, no-cache, no-store] Connection:[keep-alive] Content-Length:[859] Content-Security-Policy:[default-src 'none'] Content-Type:[application/json] Date:[Mon, 14 Oct 2024 18:01:50 GMT] Expires:[Mon, 14 Oct 2024 18:01:50 GMT] Pragma:[no-cache] Retry-After:[13] Strict-Transport-Security:[max-age=31536000] Vary:[Authorization, X-Filter] X-Accepted-Oauth-Scopes:[linodes:read_write] X-Content-Type-Options:[nosniff] X-Customer-Uuid:[---UUID HERE---] X-Frame-Options:[DENY DENY] X-Oauth-Scopes:[*] X-Ratelimit-Limit:[5] X-Ratelimit-Remaining:[3] X-Ratelimit-Reset:[1728928924] X-Spec-Version:[4.189.1] X-Xss-Protection:[1; mode=block]]
```

Reply:
Updated the PR to remove default values.